contactsummary/js/src/contact-summary/services/item.service.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-env amd */
+
+define([
+  'common/angular',
+  'common/moment'
+], function (angular, moment) {
+  'use strict';
+
+  itemService.__name = 'itemService';
+
+  function itemService () {
+    var factory = {};
+
+    /**
+     * @returns {(Object|itemService)}
+     */
+    factory.createInstance = function () {
```

Comment:
This is failing `Function Declarations to Hide Implementation Details` point of  styleguide
https://compucorp.atlassian.net/wiki/spaces/PCHR/pages/121371880/AngularJS+Style+Guide
Please check other files too, I saw this in some other `service` files too.